modules/p2/ding_serendipity_lists/ding_serendipity_lists.module

```diff
@@ -308,7 +312,8 @@ function ding_serendipity_lists_loan_adhl_serendipity_add($context) {
           if (is_array($isbn)) {
             $isbn = current($isbn);
           }
-          $recommendations = ting_get_object_recommendations($isbn, 4);
+          // TODO BBS-SAL BBS-213: Implement search provider call here.
```

Comment:
Yes please :-)

Reply:
If ADHL is terminal then I think there is little value in implementing a provider call for this.

Reply:
As mentioned in another comment :-)

It turns out that ADHL is not quite terminal yet and a ticket should be created.